docs/disaster-recovery.md

```diff
@@ -17,91 +17,48 @@ This documentation assumes that `rbd mirroring` is set up between
 For more information on how to set up rbd mirroring, refer to
  [ceph documentation](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/).
 
-## Deploy the Volume Replication CRD
+## Enable CSI-ADDONS Sidecar
```

Comment:
The name of the project is **CSI-Addons**, please use the correct capitalization.